frontend/src/components/portforward/index.tsx

```diff
@@ -187,7 +185,27 @@ export default function PortForwardingList() {
           {
             label: 'Local Port',
             getter: portforward => {
-              return portforward.port;
+              return (
+                <Box display={'flex'} alignItems="center">
+                  <Box
+                    mr={1}
+                    onClick={() => {
+                      window.open(`http://localhost:${portforward.port}`, '_blank');
+                    }}
+                    style={{ color: '#2774B3', cursor: 'pointer' }}
+                  >
+                    {portforward.port}
+                  </Box>
```

Comment:
Box components translate to divs, so we shouldn't use them for reactive components unless nothing else applies.
In this case, MUI has a Link component which is what we want to use.